connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -529,3 +555,42 @@ async function handlePptxFile(
     return null;
   }
 }
+
+async function handleCsvFile(
```

Comment:
I assume it's taken from gdrive; naive question: what is the intent in that spreadsheet & csvs go through a different flow, although they're both the same handling?

Reply:
CSVs are handled as "raw" text files while spreadsheet are returned parsed by the APIs